mikenai/gowork
# add prometheus exporter

How to check the metrics on a local machine? 
- Install [Metricat](https://metricat.dev) or Grafana stack on a local machine. The following will be using Metricat.
- Launch your server 
- Start Metricat and specify a path to metrics user like: `localhost:8080/metrics'
- Metrics related to the database pool will have a `general_main_` prefix
- Start sending requests to the API to see changes on the charts
